src/Uno.UI.RuntimeTests/Helpers/ConditionalTestAttribute.cs

```diff
@@ -19,7 +19,7 @@ public enum RuntimeTestPlatform
 }
 
 [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
-public sealed partial class ConditionalTestAttribute : TestMethodAttribute
+public partial class ConditionalTestAttribute : TestMethodAttribute
```

Comment:
Hum I didn't saw that attribute earlier ... this is not compatible with the runtime-test engine nor MS tests, we should instead implement and use (in runtime test engine, and uno) the `Execute` method on the attribute https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.testmethodattribute.execute?view=visualstudiosdk-2019#microsoft-visualstudio-testtools-unittesting-testmethodattribute-execute

Reply:
https://github.com/unoplatform/uno.ui.runtimetests.engine/issues/177

(Then we will backport to uno ... the ref must remain the runtime-test project)

Reply:
@dr1rrb I added support for this attribute in Uno implementation for runtime tests. But yeah it will be a problem when we move to uno.ui.runtimetests.engine. We already have usages of that attribute though 😕

Reply:
That was in https://github.com/unoplatform/uno/pull/16498

Reply:
The tests are so flaky in CI while passing quite well locally. So I may actually just revert the test changes for now and keep them disabled completely on Skia 😕

Reply:
Some tests are stable in CI while others are not. Anyway, I reverted the test changes for now until we do the TestMethod.Execute work